### ISSUE TITLE
zero loss issue

### ISSUE THREAD

**Issue by RyuJunHwan** [xiaohaoChen/rrc_detection#45]
The labelmap_voc.prototxt file and the value of num_classes are modified appropriately

so that the learning starts with **loss zero** and does not change.

I have also changed learning_rate, but nothing has changed.

What is the problem?

(I ran the following command.)
$ python examples/car/rrc_kitti_car.py

<img width="937" alt="1" src="https://user-images.githubusercontent.com/10956589/34347360-98f77572-ea45-11e7-9d65-dc2666edd090.png">

**Comment by xiaohaoChen** [comment]
Why you need to modify the labelmap_voc.prototxt file and the value of num_classes? Maybe you can run our code without any modification.

**Comment by RyuJunHwan** [comment]
I thought that I had to modify the file in order to proceed with learning with the desired data.
I checked the normal operation after building the new download.
Thank you for your reply.
